pkg/config.go

```diff
@@ -33,6 +33,22 @@ func (s SystemConfig) GetValue(name string) *string {
 	return &value
 }
 
+// GetValueOrEmpty returns the value in the underlying map if it exists, otherwise empty string is returned
+func (s SystemConfig) GetValueOrEmpty(name string) string {
+	value, ok := s[name]
```

Comment:
Wouldn't this already return the zero value here if the key doesn't exist?

Reply:
Also, should we actually log this? Seems like it'd be unneeded chatter.

Reply:
@rushtehrani 

1. Yep, yes it would. Will fix.

2. I'm fine with not logging it. Doesn't make sense for this method. What do you think about logging for the `GetValue` method above?